vllm/compilation/backends.py

```diff
@@ -108,6 +120,8 @@ def split_graph(graph: fx.GraphModule,
 # we share the global graph pool among all the backends
 global_graph_pool = None
 
+compilation_start_time = 0.0
```

Comment:
Maybe None instead of 0.0?

Reply:
Just wondering: can we somehow make this more robust? Since the code touching this var is scattered into different places, I feel it's error prone...

Reply:
that's why I want to merge some functions in the worker/executor. however, given the current code status, i don't have bandwidth to refactor those files.